changelog_entry.yaml

```diff
@@ -0,0 +1,4 @@
+- bump: patch
```

Comment:
Can we make this a minor and "added" instead of fixed

Reply:
make minor